site/.build.js

```diff
@@ -6,20 +6,21 @@ function getTwigTSFiles(dir) {
     return fs.readdirSync(dir, { withFileTypes: true }).reduce((acc, entry) => {
         if (entry.isDirectory()) {
             acc.push(...getTwigTSFiles(path.join(dir, entry.name)));
-        } else if (entry.name.endsWith('.ts') || entry.name.endsWith('.js')) {
+        }
+        else if (entry.name.endsWith('.ts') || entry.name.endsWith('.js')) {
             acc.push(path.join(dir, entry.name));
         }
-        
+
         return acc;
     }, []);
 }
 
 function getAllFiles(dir) {
-    return getTwigTSFiles(path.join(dir, "twig")).concat(fs.readdirSync(dir, { withFileTypes: true }).reduce((acc, entry) => {
-        if (!entry.isDirectory() && entry.name.endsWith('.ts') || entry.name.endsWith('.js')) {
```

Comment:
Just want to make sure this functionality is what is wanted, this is checking if the file is not a directory, and it either ends with `.js` or `.ts`, correct?

Reply:
Yes, that's correct, since it wouldn't make sense we accepted a directory that suffixed with '.js' when we want a js file.